### PR TITLE
Tidying up Readme and adding cover

### PR DESCRIPTION
**Pull request by gunnigylfa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# Game Chain Contracts
+# World Chain Contracts
 
-*Disclaimer: This is very much a work in progess.*
+![Project Awakening cover logo](./ProjectAwakeningCover.png)
 
-This repository contains the smart contract code supporting Project Awakening and targeted for deployment on the game chain. It contains of two separate code paths. The standard contracts are plain solidity contracts intended as support structure. The mud contracts are the functionality or *business logic* itself. 
+This repository contains the smart contract code supporting Project Awakening and targeted for deployment on the world chain. It contains of two separate code paths. The standard contracts are plain solidity contracts intended as support structure. The mud contracts are the functionality or *business logic* itself.
```
Comment (0xxlegolas) [inline README.md:5]: does world chain the right word  ?

Comment (gunnigylfa) [reply README.md:5]: Its "a" word. At least for the time being

Comment (0xxlegolas) [reply README.md:5]: can we rename to `to the world` or `on the game chain`.

Comment (gunnigylfa) [reply README.md]: Sure, renamed in aa51e59

**Pull request merged**